mandatory/4-tax.js

```diff
@@ -5,8 +5,10 @@
   Sales tax is 20% of the price of the product.
 */
 
-function calculateSalesTax() {}
-
+function calculateSalesTax(sales) {
+  const calcTax = sales * 0.2 + sales;
```

Comment:
Can you think of a way to does not requires the +?

Reply:
Thank you for making me to find other ways, This is what I like most.
So in this senario we can do this:
`const calcTax = sales * 1.2;`